data/ir-ads

```diff
@@ -0,0 +1,355 @@
+# List of ads and tracking company in Iran.
+# Thanks to PersianBlocker project
+# https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlockerHosts.txt
+
+#-------------------------v Persian Ads/Trackers/Annoyances Filters v-----------------------#
+# >> A <<
+analyt.ir
+
+scripts-ads.s3.ir-thr-at1.arvanstorage.com
+error-tracking.arvancloud.com
+
+adexo.ir
+adskav.com
+advn.ir
+
+amaroid.net
+
+analytics.aasaam.com
+analytics-2.aasaam.com
+analytics-3.aasaam.com
+
+adwised.com
+adwisedfs.com
+adwised.azureedge.net
+# TODO: Remove adwised.azureedge.net, but navatv.com and baran-music.com still links to it
```

Comment:
why not remove this site now? If it will break sites like tv or music service, then we should not use the blocking strategy based on domain.
Extensions like the ublock can do a much better job.

Reply:
Removed.